### ISSUE TITLE
Green background is showing around the video when activity with simpleexoplayerview is paused and resumed again.

### ISSUE THREAD

**Issue by PrathibhaKota** [google/ExoPlayer#3927]
![device-2018-03-02-191233](https://user-images.githubusercontent.com/22744351/36902600-0f3fe310-1e51-11e8-939d-ec3274a54b75.png)

public class MainActivity extends AppCompatActivity {

    private PlayerView playerView;
    private SimpleExoPlayer player;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        playerView = findViewById(R.id.player_view);
        preparePlayer(playerView);
    }

    public void preparePlayer(PlayerView simpleExoPlayerView){
        DefaultBandwidthMeter bandwidthMeter = new DefaultBandwidthMeter();
        TrackSelection.Factory adaptiveTrackSelectionFactory =
                new AdaptiveTrackSelection.Factory(bandwidthMeter);
        DefaultTrackSelector trackSelector = new DefaultTrackSelector(adaptiveTrackSelectionFactory);

        player = ExoPlayerFactory.newSimpleInstance(this, trackSelector);

        simpleExoPlayerView.setPlayer(player);
        Uri mp4Uri = Uri.parse("https://graph.xploree.in/img/dev_card/end_of_the_line.mp4");

        MediaSource videoSource1 = new ExtractorMediaSource(mp4Uri, buildDataSourceFactory(null), new DefaultExtractorsFactory(),
                null, null,null);

        player.prepare(videoSource1);
        player.setPlayWhenReady(true);

    }

    public DataSource.Factory buildDataSourceFactory(DefaultBandwidthMeter bandwidthMeter) {
        return new DefaultDataSourceFactory(this, bandwidthMeter,
                buildHttpDataSourceFactory(bandwidthMeter));
    }

    public HttpDataSource.Factory buildHttpDataSourceFactory(DefaultBandwidthMeter bandwidthMeter) {
        return new DefaultHttpDataSourceFactory(Util.getUserAgent(this, "videoplaysample"), bandwidthMeter);
    }

    @Override
    protected void onPause() {
        super.onPause();
        if(player != null){
            player.setPlayWhenReady(false);
        }
    }

    @Override
    protected void onResume() {
        super.onResume();
        if(player != null){
            player.setPlayWhenReady(true);
        }
    }
}

<?xml version="1.0" encoding="utf-8"?>
<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:orientation="vertical"
    tools:context="com.example.pkota.videoplay.MainActivity">

    <com.google.android.exoplayer2.ui.PlayerView
        android:id="@+id/player_view"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        app:surface_type="texture_view"
        />

</FrameLayout>
[VideoPlay.zip](https://github.com/google/ExoPlayer/files/1775319/VideoPlay.zip)

**Comment by erdemguven** [comment]
Duplicate of #2321

**Comment by erdemguven** [comment]
Please provide complete information as requested in the issue template. The issue template can be found [here](https://github.com/google/ExoPlayer/blob/release-v2/ISSUE_TEMPLATE).

**Comment by ram992** [comment]
@erdemguven, is this really an exo-player bug, cause I have been seeing the same issue, but with a blue background, that I have not used in my app at all.
And also the backGround appears on touching the empty surface that I pass to exoPlayer in the initializing phase of the Video, it won't happen afterwards.

**Comment by PrathibhaKota** [comment]
Link to test content:
http://www.sample-videos.com/video/mp4/480/big_buck_bunny_480p_2mb.mp4
In all the devices it is working fine but the issue happens only in specific devices like Coolpad Y83, Panasonic Eluga Ray X. 
I have tried with .mpd url as below
http://demos.webmproject.org/dash/201410/vp9_glass/manifest_vp9_opus.mpd.
this green color background is not happening with the mpd url even for Coolpad Y83 and Panasonic.

Device(s) and version(s) of Android being used:
Coolpad Y83 -- 6.0
Panasonic Eluga Ray X -- 6.0

Version of Exoplayer:
 Latest Exoplayer 2.7.0 version.

Reproduction steps:
1. While video is playing, press the Home button.
2.Resume the video by selecting app from Recents app list
3.Repeat the above steps, then Green background is displaying and video is displaying on the top left corner.
Note:  i cannot release the player and prepare the player again, as i have to resume the video from previously paused position.

**Comment by pavelcoder** [comment]
Users reported about same issue in Google Play feedbacks of my app. 
Bug appears when switching to fullscreen and activity is recreated
Exoplayer 2.6.1 + HLS.
Devices: 
Maizu M5S - three times
Doogee x5 max
Doogee x7 pro
Doogee Y300 - two times
ZTE BLADE A510
BQ BQS-5020 Strike

Problem occurs only in Android 6.0.

**Comment by ram992** [comment]
@PrathibhaKota, you said that you could not release and recreate the player, if previous postion is the only problem then,
you can save the current position in the onPause callBack, and in onResume create the player and then apply the time to it.  You can apply that by seeking the player once it is ready, Most of the apps does this.

**Comment by tonihei** [comment]
> Note: i cannot release the player and prepare the player again, as i have to resume the video from previously paused position. 

Do you keep the player (including the decoders) while the app is minimized? Please release the player in onPause for API <= 23 or onStop for API > 23 similar to [how our demo app is doing that](https://github.com/google/ExoPlayer/blob/release-v2/demos/main/src/main/java/com/google/android/exoplayer2/demo/PlayerActivity.java#L232).
